README.md

```diff
@@ -9,37 +9,53 @@ open source packages, container images, and CI/CD configurations to identify mis
 ## Example usage for IaC and SCA
 
 ```yaml
-on: [push]
+name: checkov
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" branch
+  push:
+    branches: [ "main", "master" ]
+  pull_request:
+    branches: [ "main", "master" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
I'd cut this as it's just a GHA mechanism

Reply:
let's leave it to make it easier for users to adapt and understand what this config does, even it has nothing to do with our action